CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+
+When installing mods without using collections, the "My Mods" screen will allow you manage your load order via the "Rules" tab.
+
+When using multiple collections, the "All" section under "Installed Mods" will allow you to manage your load order via the "Rules" tab. 
+
+This tab will be expanded in future to include other load-order or file-order rules related to the mods in your loadout or collection. 
+
+## More Features
+* Updated the Health Check messages for Cyberpunk 2077 to match the structure of other diagnostic messages.
+* When hovering the cursor to resize panels, a solid line will now appear to better indicate which panel will be adjusted.
+* The app now has its own minimise, maximise and close buttons, removing the operating system "chrome" and saving more vertical space. 
+* The tab bar is now hidden until there are at least two tabs in a panel.
+* Tabs can now be closed with a middle click.
+* Added mod thumbnails to the load order page.
+* Increased resilience to network errors during downloads.
+
+## Technical Changes
+* On Linux, the app will only register itself as the default Nexus Mods download handler when not installed with a package manager.
+* Replaced the Stardew Valley preview UI elements with game-agnostic versions.
+* Added the core backend frameworks required for the "Undo" feature.
+* Added the core backend frameworks to allow the app to create and share collections.
+* Improved the stability of downloads by requesting the entire file rather than using range requests.
+* Use the recent rating for collections instead of "All time".
+* The app will now send headers with Nexus Mods requests to allow us to better track mod manager usage.
+
+## Bug Fixes
+* Games can be unmanaged via the command line (thanks to Michael-Kowata on GitHub).
+* Fixed missing colours to represent added, removed, or changed files on the Preview Changes page.
+* Fixed an issue where the app could end up in a state where the Apply button was not visible but the Launch button was disabled.
+* Fixed an issue where the app would unnecessarily backup files during an Apply, leading to long processing times. 
+* Fixed a rare issue where the Apply action would not work correctly if two files have the same minimal hash.
+* Fixed a crash when snapping to full screen with multiple panels open.
+* Fixed "No game installed with active loadout" error that occurred when a loadout had not been applied since managing the game.
+* When un-managing a game, loadouts for that game will be hidden to prevent a crash when trying to access them.
+* Fixed a bug where copying a loadout could result in two loadouts with the same name.
+* Fixed a bug where the app may incorrectly treat Premium users as free users for collection downloads if their session expires right as the download starts.
```

Comment:
Technically the session did not expire, as in the 6 hours of the token were not up.
It was an issue with how the App cached token info, it'd happen after an hour.

And it also was not just collection downloads, there were potential for side effects elsewhere, e.g. in the top bar.

So just along the line of 'Fixed a bug where parts of the App would treat Premium users as free users after an hour of the App being launched' would be more accurate here.